components/omnibar/signed-in/signed-in.hbs

```diff
@@ -0,0 +1,226 @@
+            <span class="logo"></span>
+          </a>
+          <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
+          <span class="portal-title">
+            App
+          </span>
+
+          <ul class="nav global-icons">
+            <li class="nav-item dropdown mega-dropdown">
+              <a
+                class="nav-link dropdown-toggle mega-dropdown-toggle"
+                href=""
+                role="button"
+                data-toggle="dropdown"
+                aria-haspopup="true"
+                aria-expanded="false"
+                title="St. Jude Cloud Mega Menu"
+              >
+                <div class="nav-menu-icon"></div>
+              </a>
+              <div
+                class="dropdown-menu mega-dropdown-menu"
+                aria-labelledby="mega-dropdown-toggle"
+                id="dropdown-main"
+              >
+                <div class="container mega-menu-container">
+                  <div class="row">
+                    <div class="col-md-4 order-md-1 mega-dropdown-col">
+                      <div class="mega-menu-title">
+                        Research Domains
+                      </div>
+                      <ul class="list-unstyled">
+                        <li class="domain pediatric-cancer">
+                          <a href="https://stjude.cloud/research-domains/pediatric-cancer">
+                            Pediatric Cancer
+                          </a>
+                        </li>
+                        <li class="domain cancer-survivorship">
+                          <a href="https://stjude.cloud/research-domains/cancer-survivorship">
+                            Cancer Survivorship
+                          </a>
+                        </li>
+                        <li class="domain non-cancerous-diseases">
+                          <a href="https://stjude.cloud/research-domains/non-cancerous-diseases">
+                            Non-Cancerous Diseases
+                            </a>
+                        </li>
+                      </ul>
+                    </div>
+                    <div class="col-md-5 order-md-2 mega-dropdown-col">
+                      <div class="mega-menu-title">
+                        Apps
+                      </div>
+                      <ul class="list-unstyled">
+                        <li>
+                          <a
+                            href="https://platform.stjude.cloud"
+                            class="app-title"
+                          >
+                            <img
+                              class="app-title__icon"
+                              src="{{ path '/images/icon-genomics-platform.png' }}"
+                              alt="Genomics Platform Icon"
+                            />
+                            <span>
+                              Genomics Platform
+                              <div class="app-subtext">Data &amp; Tools</div>
```

Comment:
If we add the app's subtitles, the one for Genomics Platform should be updated to "Sequencing Data and Analysis" rather than using our carry-over language from before the redesign.